pyiceberg/catalog/rest.py

```diff
@@ -734,9 +739,11 @@ def _commit_table(self, table_request: CommitTableRequest) -> CommitTableRespons
             CommitFailedException: Requirement not met, or a conflict with a concurrent commit.
             CommitStateUnknownException: Failed due to an internal exception on the side of the catalog.
         """
+        identifier = TableIdentifier(namespace=table.identifier[1:-1], name=table.identifier[-1])
```

Comment:
Would using `self._identifier_to_tuple_without_catalog` until the use of catalog name is deprecated in 0.9.0 be safer than removing the first identifier in all cases?

Reply:
Thanks, yes, this was still work in progress 🙈